src/khoj/routers/web_client.py

```diff
@@ -364,3 +365,56 @@ def computer_config_page(request: Request):
             "khoj_version": state.khoj_version,
         },
     )
+
+
+@web_client.get("/view/{public_conversation_slug}", response_class=HTMLResponse)
```

Comment:
Should we call it `/public/chat`? It will generalize better to public agents and automations later as well

```suggestion
@web_client.get("/public/chat/{public_conversation_slug}", response_class=HTMLResponse)
```

Reply:
How about `share` instead? Nice and simple.